qdrant-landing/content/documentation/embeddings/gemini.md

```diff
@@ -38,52 +38,63 @@ The following example shows how to embed a document with the `models/embedding-0
 ## Embedding a document
 
 ```python
-import pathlib
-import google.generativeai as genai
-import qdrant_client
+import google.generativeai as gemini_client
+from qdrant_client import QdrantClient
+from qdrant_client.http.models import Distance, PointStruct, VectorParams
+collection_name = "example_collection"
 
 GEMINI_API_KEY = "YOUR GEMINI API KEY"  # add your key here
 
-genai.configure(api_key=GEMINI_API_KEY)
+gemini_client.configure(api_key=GEMINI_API_KEY)
+texts = [
+    "Qdrant is the best vector search engine!",
+    "Loved by Enterprises and everyone building for low latency, high performance, and scale.",
+]
 
-result = genai.embed_content(
-    model="models/embedding-001",
-    content="Qdrant is the best vector search engine to use with Gemini",
-    task_type="retrieval_document",
-    title="Qdrant x Gemini",
-)
+results = [
+    gemini_client.embed_content(
+        model="models/embedding-001",
+        content=sentence,
+        task_type="retrieval_document",
+        title="Qdrant x Gemini",
+    )
+    for sentence in texts
+]
 ```
 
-The returned result is a dictionary with a key: `embedding`. The value of this key is a list of floats representing the embedding of the document.
+## Creating Qdrant Points and Indexing documents with Qdrant
+
+### Creating Qdrant Points
 
-## Indexing documents with Qdrant
+```python
+points = [
+    PointStruct(
+        id=idx,
+        vector=response['embedding'],
+        payload={"text": text},
+    )
+    for idx, (response, text) in enumerate(zip(results, texts))
+]
+```
 
 ```python
-from qdrant_client.http.models import Batch
-
-qdrant_client = qdrant_client.QdrantClient()
-qdrant_client.upsert(
-    collection_name="GeminiCollection",
-    points=Batch(
-        ids=[1],
-        vectors=genai.embed_content(
-            model="models/embedding-001",
-            content="Qdrant is the best vector search engine to use with Gemini",
-            task_type="retrieval_document",
-            title="Qdrant x Gemini",
-        )["embedding"],
-    ),
+search_client.create_collection(collection_name, vectors_config=
+    VectorParams(
+        size=768,
+        distance=Distance.COSINE,
+    )
 )
+search_client.upsert(collection_name, points)
```

Comment:
I can't find this code on the preview

It seems that there is a problem with code cells

the highlighted one is not clickable

<img width="967" alt="image" src="https://github.com/qdrant/landing_page/assets/22641570/6228314c-f01d-4e8b-a401-86f03845ffff">